packages/terra-responsive-element/src/breakpoints.js

```diff
@@ -0,0 +1,30 @@
+import '../src/breakpoints.scss';
```

Comment:
Nice catch, this may or may not have been initially written a long time ago in a galaxy where scss wasn't yet compiled into the lib folder.